docs/examples/llama_dataset/uploading_llama_dataset.ipynb

```diff
@@ -87,13 +87,12 @@
    "metadata": {},
```

Comment:
<p>nit better remove those logs. since it is too long</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>good call!</p>